sycl/test-e2e/Basic/built-ins/host_math.cpp

```diff
@@ -34,7 +34,24 @@ void testCospi() {
   assert(hostDifference <= differenceExpected && "Host result incorrect");
 }
 
+void testRemquo() {
+  {
+    int quo = 0;
+    float rem = sycl::remquo(86.0f, 10.0f, &quo);
```

Comment:
@0x12CC I know I provided this in my reproducer, but this is not a SYCL 2020 conformance code.
In SYCL 2020, sycl::remquo requires second arg as genintptr, and genintptr is

> All permutations of multi_ptr<DataT, AddressSpace, IsDecorated> where DataT is all types within genint, AddressSpace is access::address_space::global_space, access::address_space::local_space and access::address_space::private_space and IsDecorated is access::decorated::yes and access::decorated::no.

and not a raw pointer. Can you fix this?

Reply:
The SYCL2020-conformant interface is already supported.